test/automatic_differentiation_test.jl

```diff
@@ -3,6 +3,7 @@ using Test
 using LinearAlgebra
 import DualNumbers: value
 import NaNMath
+import Base: one, isone, zero, iszero
```

Comment:
These are exported from base, so this is unnecessary.
```suggestion
```

Reply:
You need to explicitly import it to extend it. Try this.

struct ABC end
one(x::ABC) = 1

Will give you the error 

ERROR: error in method definition: function Base.one must be explicitly imported to be extended
Stacktrace:
 [1] top-level scope at none:0

Reply:
I know that, but they are not being extended in the tests.